deployment/ansible/roles/nih-wayfinding.app/tasks/static-files.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Create folders for static files
+  file: path={{ item }}
+        state=directory
+        owner=root
+        group=nih-wayfinding
+        mode=0775
+  with_items:
+    - "{{ app_static_root }}"
+    - "{{ app_media_root }}"
+    - "{{ app_static_cache }}"
+
+- name: Install Sass
```

Comment:
We got bit by issues that we thought were associated with `gulp-sass`, and that led us to use the Ruby Sass. If the Grunt plugin for Sass doesn't cause problems, you may be able to remove this dependency (and the associated Ruby runtime install).